logtracer/_global_vars.py

```diff
@@ -0,0 +1,4 @@
+"""Variables to be used across tracing and logging. These are set by jsonlog.configurelogging.py."""
```

Comment:
> sometimes we define global vars in settings.py 
I avoided `settings.py` as a name as I feel this implies a user can/should change the values directly, when in fact they shouldn't. I can make this clearer if that would help?

> (using uppercase)
These variables are set by the functions `configure_tracing` and `configure_json_logging`, I feel that they shouldn't be upper case because upper case means constant, and these change on running a function.

> directly in the module using them.
These vars are used in both `jsonlog` and `tracing` so I thought it would make sense for the variables to live outside of them

Reply:
ok, makes sense as is! :)

Reply:
I may be going mad, but `configurelogging.py` doesn't seem to exist any more?

Reply:
> I avoided settings.py as a name as I feel this implies a user can/should change the values directly, when in fact they shouldn't.

I'd still be more comfortable passing these in as environment vars if we can?

Reply:
Why the underscore in `_global_vars.py`? Is this a convention?

Reply:
If this is being used as a library, could we accept these as parameters rather than getting them from inside our code?